fs/spiffs/src/spiffs_vfs.c

```diff
@@ -1187,7 +1187,10 @@ static int spiffs_fstat(FAR const struct file *filep, FAR struct stat *buf)
     }
 
   spiffs_unlock_volume(fs);
-  return spiffs_map_errno(ret);
+
+  ret = spiffs_map_errno(ret);
+
+  return ret > 0 ? OK : ret;
```

Comment:
Shouldn't it be: 
```
  return ret >= 0 ? OK : ret;
```

Reply:
Yes, I think if should check for >= 0 see:

```
return spiffs_map_errno(OK);
```

Reply:
> Shouldn't it be:
> 
> ```
>   return ret >= 0 ? OK : ret;
> ```

It's the same, right? Because if it's equal to `0`, it will return `0` (a.k.a `OK`). Am I missing something here?

Reply:
Yes! I was suggesting just for semantic logic, because in theory the else ":" was just for negative values

Reply:
Done